apps/wallet/src/ui/app/shared/faucet/FaucetRequestButton.tsx

```diff
@@ -10,17 +10,26 @@ import { useFaucetRateLimiter } from './useFaucetRateLimiter';
 import { API_ENV_TO_INFO } from '_app/ApiProvider';
 import { Button, type ButtonProps } from '_app/shared/ButtonUI';
 import { useAppSelector } from '_hooks';
+import { API_ENV } from '_src/shared/api-env';
 
 export type FaucetRequestButtonProps = {
 	variant?: ButtonProps['variant'];
 	size?: ButtonProps['size'];
 };
 
+const FAUCET_HOSTS = {
+	[API_ENV.local]: getFaucetHost('localnet'),
+	[API_ENV.devNet]: getFaucetHost('devnet'),
+	[API_ENV.testNet]: getFaucetHost('testnet'),
+};
```

Comment:
I could use some help figuring out what we want to do here.  Since we are not going to have faucet methods on the SuiClient, we can't just use the rpcClientProvider.  This would work for the 3 networks listed above, and I'm not sure the wallet currently has a way to use a faucet with a custom network, but its not really an ideal solution